snapshot-labs/sx-monorepo
# fix: add validation to whitelist input

### Summary



Closes: #121

- Add validation to whitelist input
- Accept `,` as delimiter, in addition of exisiting newline

### How to test

1. Go to http://localhost:8080/#/create
2. Move to Voting power steps
3. Add a "Whitelist" strategy
4. Try inputting some invalid and valid values in the "whitelist" input
5. It should show an error when the input is not valid 
6. Create the space
7. It should create the space without errors


### Notes 

- The input should accept only format `ADDRESS:VP`
- Multiple values can be delimited by a new line and/or by `,`
- Empty lines should be ignored
- Blank spaces should be ignored (`ADDRESS:VP   `, `ADDRESS : VP` are both valid
